### PR TITLE
feat: Add edgeColorColumn to DataGraph

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [ ] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
@GordonSmith please review

**Comment by jbrundage** [comment]
image

File: packages/graph/src/graph2/graph.ts [hunk 1]
```diff
@@ -683,6 +683,9 @@ export class Graph2 extends SVGZoomWidget {
                     .on("mouseout", function (d) {
                         context.edge_mouseout(d3Select(this), d);
                     })
+                    .style("stroke", d => {
+                        return d.props?.color ?? this.edgeColor();
+                    })
```
Comment (GordonSmith) [inline packages/graph/src/graph2/graph.ts:688]: This should be in the "update" section not the enter "section"

File: packages/graph/src/graph2/graph.ts [hunk 2]
```diff
@@ -701,7 +704,9 @@ export class Graph2 extends SVGZoomWidget {
                     })
                     .remove()
             )
-            .style("stroke", this.edgeColor())
+            .style("stroke", d => {
+                return d.props?.color ?? this.edgeColor();
+            })
```
Comment (GordonSmith) [inline packages/graph/src/graph2/graph.ts:709]: This should be sufficent?

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Review by GordonSmith — CHANGES_REQUESTED** [review]
This is also failing to build.

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**